source/advanced_components/ddc/usage.rst

```diff
@@ -355,7 +355,7 @@ All deployment steps (create, provision, configuration) are covered by a single
 
 Deployment of a new provision is a 4 step process:
 
-- **Add**. OpenNebula provision objects (cluster, hosts, datastores, networks) are created, but disabled for general use.
+- **Add**. OpenNebula infrastructure objects (cluster, hosts, datastores, networks) are created, but disabled for general use.
 - :ref:`Provision <ddc_provision>`. Resources are allocated on the remote provider (e.g. use the provider's API to get clean new hosts).
 - :ref:`Configure <ddc_config>`. Resources are reconfigured for a particular use (e.g. install virtualization tools on new hosts).
 - **Enable**. Ready-to-use resources are enabled in OpenNebula.
```

Comment:
Below in the table of parameters, we are missing new ones for
-  wait & timeout
- skip (provision / config)